error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/FileRulesTestInput.java

```diff
@@ -39,4 +39,9 @@ ImmutableSet<File> testFilesCreateTempFileToFile() throws IOException {
   File testFilesCreateTempFileInCustomDirectoryToFile() throws IOException {
     return File.createTempFile("foo", "bar", new File("baz"));
   }
+
+  boolean testMkdirsBeforeFilesExists() {
+    Path foo = Path.of("foo");
```

Comment:
We generally don't create local variables in tests. I can see how in this case it _would_ make sense. However, I see that it still works if we inline it and create a `Path` twice. Now I'm not 100% what we prefer, as officially it should be the same variable. 

One of the other similar case is where we have e.g. `CheckElementIndexWithMessage`. Will go with the approach we took there and therefore inline the variable.